### DISCUSSION TITLE
修改 caddy 代理的 443 和 80 端口

### DISCUSSION THREAD

**Discussion by Jengro777** [admpub/nging#701] [General]
如果需要 https ，在自家网络里是不能通过 443 和 80 端口的。 caddy 支持自定义 http 和 https 的端口，希望这个功能可支持可视化配置

**Comment by Jengro777** [comment]
{
  http_port 800
  https_port 4433
}

**Reply by admpub** [reply]
用自动更新的 let's encrypt 不支持指定非80和443的端口，无法通过验证。如果你自己手动设置的证书可以在指定域名的时候带上网址的scheme和端口就行，比如: `http://coscms.com:800 https://coscms.com:4433`
